qemu/tests/cfg/blockdev_commit_non_existed_base.cfg

```diff
@@ -0,0 +1,36 @@
+- blockdev_commit_non_existed_base:
+    type = blockdev_commit_non_existed_base
+    virt_test_type = qemu
+    only Linux
+    images += " data"
+    force_create_image_data = yes
+    force_remove_image_data = yes
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 500M
+    image_name_data = data
+    snapshot_tags_data = sn1 sn2 sn3 sn4
```

Comment:
it might not need to create so many snapshots, only one is enough for this case, then do commit with top=sn1, base=sn0, we should get the same error msg. That's because tc will always create a file after each snapshot creation, and for this case that's not required